ProcessLib/LIE/HydroMechanics/HydroMechanicsProcess.cpp

```diff
@@ -234,30 +311,57 @@ void HydroMechanicsProcess<GlobalDim>::initializeConcreteProcess(
         }
         _process_data.mesh_prop_b = mesh_prop_b;
 
-        auto mesh_prop_k_f = const_cast<MeshLib::Mesh&>(mesh).getProperties().template createNewPropertyVector<double>("k_f", MeshLib::MeshItemType::Cell);
+        auto mesh_prop_k_f = const_cast<MeshLib::Mesh&>(mesh)
+                                 .getProperties()
+                                 .template createNewPropertyVector<double>(
+                                     "k_f", MeshLib::MeshItemType::Cell);
         mesh_prop_k_f->resize(mesh.getNumberOfElements());
         _process_data.mesh_prop_k_f = mesh_prop_k_f;
+
+        auto mesh_prop_fracture_stress_shear =
+            const_cast<MeshLib::Mesh&>(mesh)
+                .getProperties()
+                .template createNewPropertyVector<double>(
+                    "f_stress_s", MeshLib::MeshItemType::Cell);
```

Comment:
subscripts "s" and "n" indicate direction in the local coordinates, like x, y, z in global coordinates.